### DISCUSSION TITLE
What's the proper syntax to rewrite sync update query, for async calls?

### DISCUSSION THREAD

**Discussion by ghevge** [sqlalchemy/sqlalchemy#6837] [Usage Questions]
I have the following sqlalchemy query:

```
new_label_data = self.request_body_json
self.db.query(PrebuiltDomainLabel)\
                   .filter(id==label_id)\
                   .update(new_label_data)
self.db.commit()
```

What's the proper way of rewriting it for async mode? 

Thanks

**Comment by CaselIT** [comment]
Hi,
I would suggest reading the 1,4/2.0 tutorial here https://docs.sqlalchemy.org/en/14/tutorial/index.html
In particular https://docs.sqlalchemy.org/en/14/tutorial/orm_data_manipulation.html mentions how to use orm enabled updates
